appveyor.yml

```diff
@@ -0,0 +1,44 @@
+max_jobs: 1
+
+image: Visual Studio 2019
+
+cache:
+  - '%LocalAppData%\NuGet\v3-cache'
+
+nuget:
+  account_feed: false
+  project_feed: false
+  disable_publish_on_pr: true
+
+install:
+  - cmd: dotnet tool restore
+
+build_script:
+  - cmd: dotnet cake --target=Full
+
+test: off
+
+artifacts:
+  - path: outputs\*.nupkg
+    name: packages
+  - path: outputs\*.snupkg
+    name: symbols
+  - path: outputs\tests\report
+    name: report
+    type: zip
+
+deploy:
+  - provider: Environment
+    name: MyGet EMG Public
+    on:
+      branch: master
+  - provider: Environment
+    name: NuGet EMG
+    on:
+      branch: master
+      appveyor_repo_tag: true
+  - provider: Environment
+    name: EMG GPR
+    on:
+      branch: master
+      appveyor_repo_tag: true
```

Comment:
Support for publishing the package to GitHub Package Registry, only when we apply a tag on master

Reply:
What about "main"?

Reply:
Fixed now :)